lib/msf/base/simple/buffer.rb

```diff
@@ -25,6 +25,8 @@ def self.transform(buf, fmt = "ruby", var_name = 'buf')
       when 'raw'
       when 'num'
         buf = Rex::Text.to_num(buf)
+      when 'hex'
+        buf = Rex::Text.to_hex(buf, '') + "\n"
```

Comment:
jeah i just added it so the output contains a newline to look better. I looked for a way to implement it in msfvenom but it adds a 0a character to raw/binary payloads when using > outfile so i removed it for now